README.md

```diff
@@ -40,6 +40,10 @@ However, using the crate on stable compilers is possible by disabling `alloc`. I
 will require the `std` feature, because stable compilers have neither `Box` nor `Arc` in no-std
 mode.
 
+### Nightly needed for `unsafe-pinned` feature
+
+This feature enables the `PinInitable` implementation on the unstable `core::pin::UnsafePinned` type. This requires the [`unsafe_pinned` unstable feature](https://github.com/rust-lang/rust/issues/125735) and therefore a nightly compiler. Note that this feature is not enabled by default.
+
```

Comment:
I'm using `cargo rdme` to generate the readme file, so please also add this in the crate level docs in `src/lib.rs`.